src/main.ts

```diff
@@ -222,7 +214,7 @@ function main(args: string[]) {
   }
 
   if (settings.externsPath) {
-    fs.writeFileSync(settings.externsPath, jsFiles[internalExternsFileName]);
+    fs.writeFileSync(settings.externsPath, externs);
```

Comment:
It might be worth just making it an argument, that way there is no confusion about it being required.